tests/end_to_end/test_linea_api.py

```diff
@@ -210,3 +210,118 @@ def test_delete_artifact_all(execute):
     assert res.values["num_versions"] == 0
     with pytest.raises(KeyError):
         assert res.artifacts["x"]
+
+
+def test_pipeline_simple(execute):
+
+    c = """import lineapy
+lineapy.create_pipeline([], "x", persist=True)
+x = lineapy.get_pipeline("x")"""
+    res = execute(c, snapshot=False)
```

Comment:
i'm wondering if we should disallow this case. although i do see functions in the future that would let you add upstream or downstream artifacts. even in that case, creating a pipeline with no artifacts should be disallowed?